src/exceptions.cc

```diff
@@ -179,11 +182,13 @@ Local<Value> WinapiErrnoException(Isolate* isolate,
   }
   Local<String> message = OneByteString(env->isolate(), msg);
 
+  v8::NewStringType type = v8::NewStringType::kNormal;
```

Comment:
If you do decide to create new variables for these, can you name the more expressively (e.g. `new_string_type` or maybe `string_type`)?